src/components/feedback/Toast/useToast.ts

```diff
@@ -32,22 +32,27 @@ const useToast = () => {
         [equals('Zoom'), always(Zoom)],
         [T, always(Slide)]
       ])
-      const options = { autoClose, transition: getTransitionType(transitionType), position, className: toastClasses }
+      const internalOptions: ToastifyToastOptions = {
```

Comment:
the renaming is unnecessary, you can leave it as it is: "options"

Reply:
I left the options parameter as is, because you can't destructure an optional parameter. So it is now necessary.

Reply:
You have several options to destructure optional values in JavaScript:
1. using the logical [OR operator ||](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR) (which you already used)
2. using the [defaultTo](https://ramdajs.com/docs/#defaultTo) ramda function (which is much faster than javascript logical OR operator and I recommend using it instead)
3. setting a default empty object value to your argument

In your case, you can apply the 3rd option, meaning you need to set a default value to your `options` argument. Like this: 
```javascript 
{ transitionType, autoClose, ...restOptions } = {} as ToastOptions
```